microbench/src/main/java/io/netty/buffer/ByteBufAccessBenchmark.java

```diff
@@ -146,12 +147,10 @@ public ByteBuf setLong() {
     }
 
     @Benchmark
-    public int readBatch() {
+    public void readBatch(Blackhole bh) {
         buffer.readerIndex(0).touch();
-        int result = 0;
         for (int i = 0, size = batchSize; i < size; i++) {
-            result += buffer.readByte();
+            bh.consume(buffer.readByte());
```

Comment:
I do not agree on this change: the purpose of NOT using BlackHole::consume is to allow vectorizations and reordering of `readByte` calls due to correct uses of checkAccess or bound-checks.

Reply:
I think we should add a comment to explain why we do what we do @franz1981 ... WDYT ?

Reply:
That's a good point!!!

Reply:
@franz1981 fancy doing a PR ?

Reply:
Count on me!

Reply:
I've removed this particular change from the changeset.